front/SettingsForm.tsx

```diff
@@ -90,6 +92,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="email"
               placeholder="Email"
+              data-cy="settings-Email"
```

Comment:
Use consistent naming for data-cy attributes. Change 'settings-Email' to 'settings-email'.

Reply:
done